Configure.pl

```diff
@@ -398,6 +398,7 @@ sub uniq {
 $config{ldlibs} = ' -lubsan ' . $config{ldlibs} if $args{ubsan} and $^O ne 'darwin';
 $config{ldlibs} = ' -ltsan ' . $config{ldlibs} if $args{tsan} and $^O ne 'darwin';
 $config{ldlibs} = $config{ldlibs} . ' -lzstd ' if $config{heapsnapformat} == 3;
+$config{ldlibs} = $config{ldlibs} . ' -latomic ' if $config{archname} =~ /mips/; # needed for mimalloc
 # macro defs
```

Comment:
This isn't correct, as we also need this on (at least) arm32. The PPC machine capable of building 32 bit is down, so i can't check it, but it might need it too. x86 doesn't (I build `-m32` on x86_64). It also looks like Solaris needs a support library for C11 atomics...

I'd prefer to go the way of probing. I've started on https://github.com/nwc10/MoarVM/pull/new/C11-mimalloc-swings-and-roundabouts but it's getting late.